services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/resource_tracker_process_messages.py

```diff
@@ -184,12 +184,16 @@ async def _process_stop_event(
     msg: RabbitResourceTrackingStoppedMessage,
     rabbitmq_client: RabbitMQClient,
 ):
+    # NOTE: MD/ANE discussed: Initial thought was to use SimcorePlatformStatus to
+    # inform RUT that there was some problem on Simcore side and therefore we will
+    # not bill the user for running the service. This is currently not used as intented
+    # therefore we will always consider it as SUCCESS for now.
+
```

Comment:
then maybe you can explain why we saw the errors in the DB? we can have a chat

Reply:
Yes, we can have a chat. But basically, when the upload to S3 failed I got an error, and based on that RUT considered the whole service run transaction as not billable (which should probably not be the case). We can discuss this in person.